frontend/src/pages/Commitment.tsx

```diff
@@ -0,0 +1,41 @@
+import { RadioButton } from "../components/ui/RadioButton";
+
+const commitment = [
```

Comment:
Maybe a separate file can be created, something like options.ts or whatever and all these arrays could be in one place